public/app/features/commandPalette/actions/dashboard.nav.actions.ts

```diff
@@ -0,0 +1,24 @@
+import { locationService, getBackendSrv } from '@grafana/runtime';
+import { Action } from 'kbar';
+
+async function getDashboardNav(parentId: string): Promise<Action[]> {
+  const data: Array<{ type: string; title: string; url: string }> = await getBackendSrv().get('/api/search');
+
+  const goToDashboardActions: Action[] = data
+    .filter((item) => item.type === 'dash-db')
+    .map((item) => ({
+      parent: parentId,
+      id: `go/dashboard/${item.url}`,
```

Comment:
Not sure if the dashboards URLs count as PII, but there is the potential of them being logged to analytics. Feel free to recommend an alternative if this is not okay.

Reply:
we already report dashboard URLs in different places (i.e. https://github.com/grafana/grafana/blob/7966e3583e680456eec831c46ceaa352f264bd0d/public/app/features/dashboard/components/SaveDashboard/useDashboardSave.tsx#L41) so it should be perfectly fine: